### ISSUE TITLE
State should not be displayed as a tree

### ISSUE THREAD

**Issue by theod** [ossia/score#126]
for now State is represented as tree : 
![capture d ecran 2016-01-27 a 16 01 09](https://cloud.githubusercontent.com/assets/118426/12617492/363f2256-c510-11e5-8b1b-42af128762ac.png)

but in the current projet I'm working on I would like to be able to send a message (/flock/add 340. 180.) several times in the same State to create many objects in a raw (yes I could create a message /flock/add/number but sometimes you can't change the code of the program you control !).

and so the current way the state is displayed doesn't allow me to deal with several instances of a same message.

finally the best gui would be a simple vertical array with 2 columns (one for the address and one for the value) : 
/flock/clear
/flock/add          340. 180.
/flock/add          30. 80.
/flock/add          40. 10.
/flock/add          30. 10.
/flock/add          40. 80.
...

one other benefit of such gui is that is easier to reorganize the order of the message to send.

**Comment by theod** [comment]
is @bltzr , @reno- , @avilleret , @jln- are agree with me ?

**Comment by jcelerier** [comment]
Originally I think that a tree was requested. Jean-Michel & Magnolya proposed to have both (see the attached pdf)
[interface_ig_iscore0.3_16.pdf](https://github.com/OSSIA/i-score/files/106935/interface_ig_iscore0.3_16.pdf)

**Comment by theod** [comment]
yes I'm agree with this : a tree of States but not a tree of Message's Address.
but I guess this is a long task ...

**Comment by reno-** [comment]
I think a list is the widget that fits our needs in this case.

An OSC message have a mandatory address, but may not have arguments.
So in a Qlistwidget, we might have 

/flock/add 10
/flock/trig
/flock/add 30
/flock/trig

What do you think about?

**Comment by theod** [comment]
it sounds good !
but I let @jcelerier confirm that is the correct solution.

**Comment by bltzr** [comment]
this is a big change and that would prevent folding nodes in the inspector's state editor, so I guess this has to be discussed (adding it to the Lyon topics)

**Comment by jcelerier** [comment]
I'm all for going from tree to list, it would simplify a lot of things: 

* Trees don't allow ordering messages by importance
* Trees don't allow having twice the same node

**Comment by bltzr** [comment]
okay
